src/contexts/Transaction/TxHistoryContext.ts

```diff
@@ -1,17 +1,23 @@
-import { providers } from 'ethers'
-import { TransactionCallbacks, TransactionLog } from 'models/transaction'
+import { TransactionCallbacks, TxStatus } from 'models/transaction'
 import { createContext } from 'react'
 
-// Prefer using tx.timestamp if tx has been mined. Otherwise use createdAt timestamp
-export const timestampForTxLog = (txLog: TransactionLog) => {
-  return (
-    (txLog.tx as providers.TransactionResponse)?.timestamp ?? txLog.createdAt
-  )
+export type TransactionType = {
```

Comment:
AFAICT, we only actually use these properties on transactions. Don't need the whole `providers.TransactionResponse`